binzx/otomi

```diff
@@ -222,6 +220,7 @@ vars=(
   PROFILE
   STATIC_COLORS
   SHELL
+  TEAM_IDS
```

Comment:
The binzx/otomi is an entry point for CLI that works with values repository
For tasks, pease create separate entry point that defines interface for task only.

Reply:
Why not use the cli? It offers advantages such as parameter parsing etc. No need to reinvent the wheel.

Reply:
There are two ways to approach this problem, my own opinion is that we should choose one approach and be consistent with it.